### PR TITLE
CRM-18092: Add batch info to bookkeeping report

### PR DESCRIPTION
**Pull request by laryn**
- [CRM-18092: Add Batch name\/title to Bookkeeping report columns](https://issues.civicrm.org/jira/browse/CRM-18092)

### CODE REVIEW

File: CRM/Report/Form/Contribute/Bookkeeping.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
  +--------------------------------------------------------------------+
  | CiviCRM version 4.7                                                |
  +--------------------------------------------------------------------+
- | Copyright CiviCRM LLC (c) 2004-2015                                |
+ | Copyright CiviCRM LLC (c) 2004-2016                                |
```
Comment (eileenmcnaughton) [inline CRM/Report/Form/Contribute/Bookkeeping.php]: Are you able to remove these lines from your change? They seem correct & probably need changing but they should be done en masse. You might want to log a JIRA to remind someone to change them...

@JoeMurray I don't really work with batches - can you eyeball the join below - or tag someone else who can?

I didn't expect batch to join like this

LEFT JOIN civicrm_entity_batch ent_batch
+                    ON  {$this->_aliases['civicrm_financial_trxn']}.id = ent_batch.entity_id AND ent_batch.entity_table = 'civicrm_financial_trxn'

but that doesn't mean it doesn't!

**Comment by eileenmcnaughton** [comment]
I guess this is vey low risk & it seems stagnated so I'm merging

**Comment by JoeMurray** [comment]
Sorry for not responding earlier, @eileenmcnaughton - the join looks fine.

**Comment by laryn** [comment]
Thanks both. I've just submitted the fix again as a PR for the 4.6 branch as well.
https://github.com/civicrm/civicrm-core/pull/7994

**Pull request merged**